frontend/packages/component-library/src/cms/image/stories/Image.story.tsx

```diff
@@ -0,0 +1,90 @@
+import type {Meta, StoryObj} from '@storybook/react';
+import {within, expect} from '@storybook/test';
+
+import Image from '../index';
+
+export default {
+  title: 'CMS/Image',
+  component: Image,
+} as Meta;
+type Story = StoryObj<typeof Image>;
+
+//
+// STORIES
+//
+export const DefaultImage: Story = {
+  args: {
+    src: 'https://images.ctfassets.net/90t6bu6vlf76/6kTQglLn3K6jP0zd54lahO/e97b9f7d9a239db2f9a8cab3238d576d/teach-page-top.png',
```

Comment:
Could we self-host this image rather than relying on Contentful?

Reply:
I used a Contentful image here since that's what this will primarily be using. When we deprecate Pegasus it'll deprecate the images we have stored there which is far off, but something to be aware of.

Reply:
I was thinking from a UI Test perspective in which a change to the asset on Contentful can break our tests. We could use a data src image or something hosted locally on Storybook, but I'm fine with us starting with this and addressing as issues crop up as well.

Reply:
Updated! Thanks for helping set up the local image hosting support 🙌